File: IQClientSite/ClientApp/src/app/IQResponses/Status/Meters.ts
import { Generator } from "./Generator";
import { Grid } from "./Grid";
import { Load } from "./Load";
import { Storage } from "./Storage";
import { Pv } from "./Pv";



export class Meters {
  last_update: number;
  soc: number;
  main_relay_state: number;
  gen_relay_state: number;
  backup_bat_mode: number;
  backup_soc: number;
  is_split_phase: number;
  phase_count: number;
  enc_agg_soc: number;
  enc_agg_energy: number;
  acb_agg_soc: number;
  acb_agg_energy: number;
  pv: Pv;
  storage: Storage;
  grid: Grid;
  load: Load;
  generator: Generator;
}

File: IQClientLib/Responses/Status/Meters.cs
using IQClientLib.Database.Models;

namespace IQClientLib.Responses.Status
{
    public class Meters 
    {
        public int Id { get; set; }
        public int last_update { get; set; }
        public int soc { get; set; }
        public int main_relay_state { get; set; }
        public int gen_relay_state { get; set; }
        public int backup_bat_mode { get; set; }
        public int backup_soc { get; set; }
        public int is_split_phase { get; set; }
        public int phase_count { get; set; }
        public int enc_agg_soc { get; set; }
        public int enc_agg_energy { get; set; }
        public int acb_agg_soc { get; set; }
        public int acb_agg_energy { get; set; }
        public Pv pv { get; set; }
        public Storage storage { get; set; }
        public Grid grid { get; set; }
        public Load load { get; set; }
        public Generator generator { get; set; }
    }




}
